build-release-tools/application/jump_version.py

```diff
@@ -70,6 +70,31 @@ def get_repo_name(self):
         repo_name = common.strip_suffix(os.path.basename(repo_url), ".git")
         return repo_name
 
+    def link_debianstatic(self):
+        """
+        Handle repository which contains debianstatic/repository_name folder,
+        for example: debianstatic/on-http
+        """
+        repo_name = self.get_repo_name()
+        linked = False
+        for filename in os.listdir(self._repo_dir):
+            if filename == "debianstatic":
+                debianstatic_dir = os.path.join(self._repo_dir, "debianstatic")
+                for debianstatic_filename in os.listdir(debianstatic_dir):
```

Comment:
it's also a dir name, not a file name, right? although dir is also a kind of files